src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/Dashboard.test.tsx

```diff
@@ -274,178 +274,6 @@ it("shows consistent counts in the chart on the fixed tab", async () => {
-  expect(
-    screen.queryByText("Turn on automatic data removal with ⁨Monitor Plus⁩"),
-  ).not.toBeInTheDocument();
-});
-
-it("shows the premium upsell dialog of the Premium upsell badge open by default)", () => {
-  const ComposedDashboard = composeStory(
-    DashboardUsNoPremiumNoScanNoBreaches,
-    Meta,
-  );
-  render(<ComposedDashboard autoOpenUpsellDialog />);
-
-  expect(
-    screen.getByText("Turn on automatic data removal with ⁨Monitor Plus⁩"),
-  ).toBeInTheDocument();
-});
-
-it("closes the premium upsell dialog of the Premium upsell badge after it opened by default)", async () => {
-  const user = userEvent.setup();
-  const ComposedDashboard = composeStory(
-    DashboardUsNoPremiumNoScanNoBreaches,
-    Meta,
-  );
-  render(<ComposedDashboard autoOpenUpsellDialog />);
-
-  expect(
-    screen.getByText("Turn on automatic data removal with ⁨Monitor Plus⁩"),
-  ).toBeInTheDocument();
-
-  const closeButtonIcon1 = screen.getByLabelText("Close modal");
-  await user.click(closeButtonIcon1.parentElement as HTMLElement);
-  expect(
-    screen.queryByText("Turn on automatic data removal with ⁨Monitor Plus⁩"),
-  ).not.toBeInTheDocument();
-});
-
-it("opens and closes the premium upsell dialog via the Premium upsell button", async () => {
-  const user = userEvent.setup();
-  const ComposedDashboard = composeStory(
-    DashboardUsNoPremiumEmptyScanResolvedBreaches,
-    Meta,
-  );
-  render(<ComposedDashboard />);
-
-  const premiumCta = screen.getByRole("button", {
-    name: "Get continuous protection",
-  });
-  expect(premiumCta).toBeInTheDocument();
-
-  await user.click(premiumCta);
-  expect(
-    screen.getByText("Turn on automatic data removal with ⁨Monitor Plus⁩"),
-  ).toBeInTheDocument();
-  const closeButtonIcon1 = screen.getByLabelText("Close modal");
-  await user.click(closeButtonIcon1.parentElement as HTMLElement);
-  expect(
-    screen.queryByText("Turn on automatic data removal with ⁨Monitor Plus⁩"),
-  ).not.toBeInTheDocument();
-});
-
-it("toggles between the product offerings in the premium upsell dialog", async () => {
-  const user = userEvent.setup();
-  const ComposedDashboard = composeStory(
-    DashboardUsNoPremiumNoScanNoBreaches,
-    Meta,
-  );
-  render(<ComposedDashboard />);
-
-  // We show a CTA on desktop in the toolbar and in the mobile menu
-  const premiumCtas = screen.queryAllByRole("button", {
-    name: "Automatic data removal: Off",
-  });
-  expect(premiumCtas.length).toBe(2);
-
-  await user.click(premiumCtas[0]);
-
-  const productTabMonthly = screen.getByRole("radio", { name: "Monthly" });
-  const productYearlyCta = screen.getByRole("link", {
-    name: "Select yearly plan",
-  });
-  expect(productYearlyCta).toBeInTheDocument();
-
-  await user.click(productTabMonthly);
-
-  const productMonthlyCta = screen.getByRole("link", {
-    name: "Select monthly plan",
-  });
-  expect(productMonthlyCta).toBeInTheDocument();
-});
-
```

Comment:
@Vinnl Yes, all this was moved to the describe `Upsell badge` block to have a better overview of which tests are related to it.